### ISSUE TITLE
Map Popup: add section address

### ISSUE THREAD

**Issue by saerdnaer** [tyrasd/overpass-turbo#179]
When selecting an object in the result map, there opens a popup which displays the tags and metadata. In an discussion on last weekend it was suggested that there should also be the option to display the address which was precalculated by nominatim for this object ID. 

I would suggest to implement this via an add an request to e.g. http://nominatim.openstreetmap.org/lookup?osm_ids=N3490627782&addressdetails=0 when the corresponding popup is opened.

**Comment by tyrasd** [comment]
Hmm. At first, this doesn't sound like a really necessary feature for overpass turbo to me. Can you point out some use-cases where this would be useful?

Maybe we could add this as a special MapCSS option… Hmm

**Comment by saerdnaer** [comment]
Yes, it's more a feature for end user maps build with overpass, than an overpass query editor feature. But I think it would reduce the number of unnecessary addr:* tags in the main api db.

Example/Use case: When I'd like to create a playground map with overpass turbo, which also lists the address of the playgrounds, users currently tend to add the full addr:* to the raw data at osm.org. As playgrounds, postboxes, ATMs etc. generally have no own address, so this user behavior is somehow not wanted.
